src/webgl/p5.Geometry.js

```diff
@@ -146,6 +150,98 @@ p5.Geometry.prototype.averagePoleNormals = function() {
   return this;
 };
 
+/**ANOTHER EXAMPLE OF HOW THIS WAS BEING DONE BEFORE**/
+// p5.Geometry.prototype._makeTriangleEdges = function() {
+//     for(var i = 0; i <= this.vertices.length; i+=3) {
+//       var i0 = i;
+//       var i1 = i+1;
+//       var i2 = i+2;
+
+//       //connections then boolean for whether it closes the triangle
+//       //also checks to see that we don't go over
+//       if(i1 <= this.vertices.length-1)
+//         this.edges[i] = [i0, i1, false];
+//       if(i2 <= this.vertices.length-1) {
+//         this.edges[i+1] = [i1, i2, false];
+//         this.edges[i+2] = [i2, i0, true];
+//       }
+//     }
+//   return this;
+// };
+
+/**THIS ACHIEVES IDENTICAL RESULTS TO ABOVE BUT RESEMBLES PROCESSING MORE**/
+p5.Geometry.prototype._makeTriangleEdges = function() {
+      for(var i = 0; i <= this.vertices.length/3; i++) {
+        var i0 = 3 * i;
+        var i1 = 3 * i + 1;
+        var i2 = 3 * i + 2;
+        if(this.vertices.length > i1)
+          this._addEdge(i0, i1, true, false);
+        if(this.vertices.length > i2)
+        {
+          this._addEdge(i1, i2, false, false);
+          this._addEdge(i2, i0, false, false);
```

Comment:
Should this ever be `false, true`? Just curious that the second boolean parameter is always false across these calls. It could just be omitted if it is intended as a constant value.